ext/DynamicExpressionsVisualizeExt.jl

```diff
@@ -0,0 +1,72 @@
+module DynamicExpressionsVisualizeExt
+
+using Plots, GraphRecipes, DynamicExpressions
```

Comment:
Is there a way to use https://github.com/JuliaPlots/Plots.jl/tree/master/RecipesBase so we don't need to directly depend on Plots.jl? If using RecipesBase.jl it can usually interface with other plotting packages.